app/routes/image-set.js

```diff
@@ -15,5 +15,32 @@ export default Ember.Route.extend( OrganizationRouteMixin, RequireUserMixin, {
 
   model: function(params) {
     return this.store.find('imageSet', params.image_set_id);
+  },
+
+  actions: {
+    associate: function(imageSet, lion) {
+      var route = this;
+      imageSet.setProperties({
+        lion: lion,
+        isVerified: false
+      });
+
+      imageSet.save().then(function() {
+        route.transitionTo('lion', lion);
+      });
+    },
+
+    createLion: function(imageSet, lionName){
+      var route = this;
+
+      var lion = this.store.createRecord('lion', {
+        primaryImageSet: imageSet,
+        name: lionName
+      });
+
+      lion.save().then(function(lion){
+        route.transitionTo('lion', lion);
+      });
```

Comment:
same Q as above about saving — does the UI give any feedback that the lion is saving?